src/lib/components/CarouselWidget.svelte

```diff
@@ -1,46 +1,55 @@
 <script lang="ts">
-	import { Carousel } from 'flowbite-svelte';
 	import type { Picture } from '$lib/types/Picture';
+	import { TinySlider } from 'svelte-tiny-slider';
+	import PictureComponent from './Picture.svelte';
+	import { onDestroy, onMount } from 'svelte';
 
 	export let pictures: Picture[];
 	export let autoplay: number;
 	let className = '';
 	export { className as class };
-	let images = pictures.map((picture) => ({
-		alt: picture.name,
-		srcset: picture.storage.formats
-			.map((format) => `/picture/raw/${picture?._id}/format/${format.width} ${format.width}w`)
-			.join(', '),
-		title: picture.name,
-		slide: picture.slider
-	}));
+	let currentIndex = 0;
+	// Function to move to the next slide
+	function nextSlide() {
+		currentIndex = (currentIndex + 1) % pictures.length;
+	}
+	const interval = setInterval(nextSlide, autoplay);
+	onDestroy(() => {
+		clearInterval(interval);
+	});
+	onMount(() => {
+		return () => clearInterval(interval);
+	});
```

Comment:
```suggestion
	onMount(() => {
		const interval = setInterval(nextSlide, autoplay);
		return () => clearInterval(interval);
	});
```

- `setInterval` should only be called in `onMount` to avoid memory leaks
- no need to have both a cleanup return function and `onDestroy`